app-benchmarks/uefi-sct/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=uefi-sct
```

Comment:
Why is it not named `edk2-test` (the name of the repository)? Is this a commonly used name?